tests/server/activity-event.js

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!object',
+  'intern/chai!assert',
+  'intern/dojo/node!bluebird',
+  'intern/dojo/node!path',
+  'intern/dojo/node!proxyquire',
+  'intern/dojo/node!sinon',
+], function (registerSuite, assert, Promise, path, proxyquire, sinon) {
+  var request, write, activityEvent;
+
+  registerSuite({
+    name: 'activity-event',
+
+    setup: function () {
+      request = {
+        headers: { 'user-agent': 'foo' },
+        query: {
+          bar: 'baz',
+          qux: 'wibble',
+          zignore: 'ignore me'
+        }
+      };
+
+      write = process.stderr.write;
+      process.stderr.write = sinon.spy();
+
+      activityEvent = proxyquire(path.resolve('server/lib/activity-event'), {
+        './configuration': {
```

Comment:
Does this still need to be mocked out? There is no `./configuration` dependency in `server/lib/activity-event.js`.